### ISSUE TITLE
Rescan an anime when trying to play its next unwatched episode

### ISSUE THREAD

**Issue by felipe5** [z411/trackma#433]
Currently when I have the last episode of an anime but the library has not been updated yet, I'll get an error "Episode not in library" if I try to play it by clicking on "Play next episode". Maybe it could be better if it  tries to scan it before sending the error, like Taiga does.

Example here trying to play episode 7:
![Screenshot_20190523_043320](https://user-images.githubusercontent.com/11513583/58239690-81635000-7d17-11e9-80bd-4dbb5f3ded43.png)

**Comment by z411** [comment]
So would you want a flow like this?

Episode in library -> Play it
Episode not in library -> Rescan Library -> Try again once

**Comment by felipe5** [comment]
> So would you want a flow like this?
> 
> Episode in library -> Play it
> Episode not in library -> Rescan Library -> Try again once

 Yes, but maybe the rescan should be for that specific anime, not the whole library.

> Or do you prefer something more complex?
> 
> As a side note, the inotify trackers should add new episodes to the library automatically, so if you're using that this shouldn't be happening.

I tried with inotify (was using pyinotify), it didn't work :(

**Comment by FichteFoll** [comment]
> As a side note, the inotify trackers should add new episodes to the library automatically, so if you're using that this shouldn't be happening.

That sounds like a useful feature regardless of which tracker was chosen.

> Yes, but maybe the rescan should be for that specific anime, not the whole library.

I imagine trackma could rescan the containing folders of each previously recognized episode of this show. The former would be a better approach, though.
